microreactors/gcmr/assembly/RIA_transient/SAM_half_tr.i

```diff
@@ -0,0 +1,259 @@
+######################################################################################################
+## Dynamic Multiphysics Modeling of Reactivity insertion accident in Gas-cooled Microreactor Assembly
+## BISON dynamic SAM half channel Model
```

Comment:
why BISON?

Reply:
Thank you for the comment. A typo was corrected in the header.